docs/docs/how-tos/nebari-environment-management.md

```diff
@@ -0,0 +1,26 @@
+---
+id: nebari-environment-management
+title: Nebari Environment Management
+description: Best Practices for Managing Your Python Environment
+---
+
+# Nebari Python Environment Management
+
+Nebari configuration and deployment is highly dependent on the version of the Nebari CLI that you have active when you run `nebari` commands.  A such, we highly recommend [installing Conda](https://docs.anaconda.com/free/anaconda/install/) for managing isolated Python environments, especially when working on more than one Nebari deployment from the same machine or using [Nebari extensions][nebari-extension-system].
+
+Once installed, you can use [manage environments](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#managing-environments) in different ways.
+
+This is a simple way to set up a Python environment with a specific Nebari version and exactly one extension installed.  *When **pip** is installed with conda, the packages will not be saved in the global namespace*.  This especially critical when working with extensions, because `nebari` will detect and apply any extensions installed in your Python environment when it runs.
+
+```
+conda create -n nebari0
+conda activate nebari0
+conda install python==3.12 pip # or python version of your choice
+pip install nebari==2024.1.1 nebari-plugin-self-registration==0.0.9
```

Comment:
``` suggestion
pip install nebari==2024.1.1 
# Plugins can also be installed 
pip install nebari-plugin-self-registration==0.0.9
```
I would split this part so that it's explicit that this is optional when running Nebari itself.